blog/2023-03-01-effective-focus-indicators/index.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: 2023/01/effective-focus-indicators
+title: Effective and flexible focus indicators
+authors: joe
+tags: [design, libraries, ui]
+---
+
+How can we design effective and flexible focus indicators for a component library?
+
+<!--truncate-->
+
+## What are focus indicators?
+
+Focus indicators help people navigate around a webpage or application using a keyboard or assistive technology. Good focus indicators make it clear what element on the page is in `:focus`. To be clear and useful, focus indicators should meet or exceed the [WCAG accessibility guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html).
+
+## Focus indicators in DHIS2-ui
+
+The DHIS2 component library, ui, offers prebuilt components for all kinds of DHIS2 applications. This helps internal and external teams build consistent user experiences.
+
+One of DHIS2-ui's guiding principles is to be flexible enough to handle all kinds of use cases. As much as possible, we don't want to limit the usefulness of the components. If a component utility is too narrow then it'll lead to more custom components and less consistency across the platform.
```

Comment:
```suggestion
One of the core guiding principles of the DHIS2 UI library is to maintain a high level of flexibility that can accommodate a wide range of use cases. Our goal is to avoid imposing unnecessary limits on the usefulness of the components. If a component's functionality is too narrowly defined, it could result in the need for more custom components, which can lead to a less consistent user experience across the platform.
```